docs/build/spells.md

```diff
@@ -0,0 +1,197 @@
+```aqua
+import Op, Debug from "@fluencelabs/aqua-lib/builtin.aqua"
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+
+func spell():
+    msg = "Spell is working!"
+    str <- Debug.stringify(msg)
+    Spell "worker-spell"
+    Spell.list_push_string("logs", str)
+```
+
+Lets modify it a bit, so that we could redirect the spell logs can be viewed though `fluence deal logs` command:
+```aqua
+import Op, Debug from "@fluencelabs/aqua-lib/builtin.aqua"
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+func spell():
+    msg = "Spell is working!"
+    str <- Debug.stringify(msg)
+    Spell "self"
+    -- also it can be resolved by Spell "spell"
+    Spell.set_string("my_key", str)
+    -- then we will log the same message to installation-spell, so it is discoverable through the CLI's `fluence deal logs` command
+    Spell "worker-spell"
+    Spell.store_log(str)
+```
+
+After that lets also modify the main.aqua file to define a helper function for getting the keys into the spell KV storage:
+```aqua
+aqua Main
+
+import Console, Debug from "@fluencelabs/aqua-lib/builtin.aqua"
+import "@fluencelabs/aqua-lib/subnet.aqua"
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+
+use "deals.aqua"
+
+export get_my_keys
+
+func get_my_keys() -> []string:
+    deals <- Deals.get()
+    dealId = deals.defaultWorker!.dealIdOriginal
+    results: *string
+    on HOST_PEER_ID:
+        subnet <- Subnet.resolve(dealId)
+    if !subnet.success:
+        Console.print(["Failed to resolve subnet: ", subnet.error])
+
+    for w <- subnet.workers:
+        if w.worker_id != nil:
+            on w.worker_id! via w.host_id:
+                Spell "1min_spell"
+                res <- Spell.get_string("my_key")
+                results <- Debug.stringify([res.str, "on worker ", w.worker_id!, " on host ", w.host_id])
+
+    <- results
+
+``` 
+
+Then lets inspect the spell.yaml file:
+```yaml
+version: 0
+
+aquaFilePath: ./spell.aqua
+
+function: spell
+
+clock:
+  periodSec: 60
+  endDelaySec: 1800
+```
+
+It contains the definition of the spell, the name of function that is triggered (in case if you don't like the default name), and the cron-ish timer based trigger configuration.
+
+- `periodSec` defines how often the spell will be executed. If set to 0, the spell will be executed only once. If this value is not provided, the spell will never be executed.
+- `endDelaySec` defines how long to wait before the last execution in seconds. If this property or `endTimestamp` is not specified, periodic execution will never end.
+
+:::info
+Your computer's clock is used to determine a final timestamp sent to the server. If it is in the past at the moment of spell creation - the spell will never be executed. This property conflicts with `endTimestamp`. You can specify only one of them.
+:::
+
+More on the spell.yaml contents can be found in:
+Documentation: https://github.com/fluencelabs/cli/tree/main/docs/configs/spell.md 
```

Comment:
Maybe paste link to a specific commit, lest it gets moved.

Reply:
Not sure how spell config could be moved. Please eleborate.

Reply:
so it is better to link to a specific commit, imo